L103-python-add-typing-to-sync-api.md

```diff
@@ -0,0 +1,79 @@
+Add Typing to gRPC Python Sync API
+----
+* Author(s): XuanWnag-Amos
+* Approver: gnossen
+* Status: Draft
+* Implemented in: Python
+* Last updated: 11/09/2022
+* Discussion at: <google group thread> (filled after thread exists)
+
+## Abstract
+
+Add type annotation to Python Sync API.
```

Comment:
Nit: "Add type annotations to the remainder of the gRPC Python API". This is almost the same thing, but makes cleat the value we intend to achieve here.

Reply:
That indeed sounds better, will change it.